utils/src/main/java/com/cloud/utils/SwiftUtil.java

```diff
@@ -294,12 +238,80 @@ public static String calculateRFC2104HMAC(String data, String key)
 
     }
 
-    public static String toHexString(byte[] bytes) {
+    static String toHexString(byte[] bytes) {
```

Comment:
What about using a commons to convert the string to Hexadecimal?
Something like: org.apache.commons.codec.binary.Hex.encode(byte[])

Reply:
sure.